cesium/internal/testdata/v1/generate.go

```diff
@@ -0,0 +1,53 @@
+package testdata
+
+import (
+	"context"
+	"github.com/synnaxlabs/cesium"
+	"github.com/synnaxlabs/cesium/internal/testdata"
+	"github.com/synnaxlabs/x/telem"
+)
+
+var (
+	ctx      = context.Background()
+	channels = testdata.Channels
+	frames   = testdata.Frames
+)
+
+func GenerateV1() error {
```

Comment:
Just keep in mind that this function will actually generate database files for the **current** version, not v1.